02_Methodiek.Rmd

```diff
@@ -242,10 +274,17 @@ $$trend - 2* sd \leq 1(cm/jaar)$$
 en 
 $$trend + 2* sd \geq -1(cm/jaar)$$
 
-Bij de modelselectie kwam toch ook nog expertoordeel kijken. 
-Sommige tijdreeksen werden bijna perfect gefit, maar omdat het aantal beschikbare metingen maar op één meetjaar sloeg, was het toch onbetrouwbaar om het model te extrapoleren over een relatief lange tijdsperiode. 
-Anderzijds werd een model met een verklarende variantie < 66% toch soms nog weerhouden, als het model goed de lage grondwaterstanden kon modelleren.
-Zo kon ook een model met een niet significante trend, maar met een heel geringe fluctuaties toch afgekeurd worden en vice versa.
+Bij de modelselectie kwam toch ook nog expertoordeel bij kijken. 
+
+Er werden geen kwaliteitseisen gesteld aan een meetreeks die deze a priori zou uitsluiten van een tijdreeksanalyse. 
+De gestelde eisen hebben alleen een weerslag op de categorie-indeling. 
+In principe kan elke peilbuis in categorie 1B terecht komen, zelfs is er maar één meting van bekend.
+En zo geschiedde ook.
+Deze meetreeksen werden manueel afgekeurd. Sommige van deze tijdreeksen hadden bijna een perfecte fit, maar het was toch onbetrouwbaar om hiermee waarden te simuleren over een relatief lange tijdsperiode. 
```

Comment:
> Er werden geen kwaliteitseisen gesteld aan een meetreeks die

> werden manueel afgekeurd.

Op een of andere manier heeft de 'manuele afkeuring' een bepaalde regel gevolgd. Wellicht kan die regel code-matig geïmplementeerd worden zodat een pre- of postselectie voor/van tijdsreeksanalysen kan gebeuren?